Modules/Chat.lua

```diff
@@ -50,8 +50,8 @@ function Chat:ShouldEnable()
 end
 
 function Chat:OnEnable()
-	self:RegisterEvent("CHAT_MSG_GROUP")
-	self:RegisterEvent("CHAT_MSG_GROUP_LEADER")
+--	self:RegisterEvent("CHAT_MSG_GROUP")
+--	self:RegisterEvent("CHAT_MSG_GROUP_LEADER")
```

Comment:
If these events were removed from the game, you can just delete these lines (and any associated handler functions) rather than commenting them out.

If new events were added to replace these (and presumably some were, unless there is no longer any distinction between a party group and a raid group) then those need to be added in their place.